### DISCUSSION TITLE
Instability problems when flying multiple crazyflies

### DISCUSSION THREAD

**Discussion by Sherry-97** [USC-ACTLab/crazyswarm#573] [Q&A]
When I was doing a multi-cfs flight, The aircraft was relatively stable during the initial takeoff phase, and I used the goto function to fly multiple coordinate points, midway through the flight the drone started to become unstable and eventually crashed. 
Then the error was reported in rviz as unable to find the position information of the cf coordinate points
I suspect it may be a problem with optitrack tracking, as I can see the warning that cf location information could not be found in a brief hover before the crash. Here are some of my setup files, hope to get some advice, thanks!
![Snipaste_2021-12-22_11-24-14](https://user-images.githubusercontent.com/70084772/147031943-dd8d4216-6aea-4f8f-a2c8-e6a966c5b1b7.png)
![Snipaste_2021-12-22_11-26-00](https://user-images.githubusercontent.com/70084772/147031950-e8c982ec-97fb-42cb-85e0-eb267dbdeff8.png)
![Snipaste_2021-12-22_11-26-20](https://user-images.githubusercontent.com/70084772/147031952-17206624-2431-4df5-bf63-ee155159c32e.png)
![Selection_011](https://user-images.githubusercontent.com/70084772/147032173-9a9f465e-4d96-4362-83be-37ad28fd968b.png)
By the way, I followed some previous advice and chose the **libobjecttracker** tracking method and the stability seems to have gotten worse, I'm not sure why

**Comment by whoenig** [comment]
This is most likely caused by object tracking being unstable (jitter or axis flips). The best way to deal with that is to use libobjecttracker with a single marker per Crazyflie. Even the multiple-marker tracking mode in libobjecttracker seems to be more robust than the tracking in Motive 2.X.

**Reply by Sherry-97** [reply]
First of all, thanks for such a timely response. 
In fact, when I tested with libobjecttracker, the results were significantly worse than motioncapure, if I added multiple markers points for each Crazyflie. Just setting one marker for each Crazyflie, which means it is impossible to create rigid bodies?（At least three marker points are required to create a rigid body）So how should  transfer the data of Crazyflie through single marker? 
Also, when I select libobjecttracker mode, do I need to make changes to the settings in Motive?

**Reply by whoenig** [reply]
Whenever you use `libobjecttracker`, you should not use any rigid bodies ('assets') at all. In Motive, make sure you delete or disable all assets. In the streaming pane, stream labeled/unlabeled markers.

libobjecttracker tracks the markers frame-by-frame (using the position provided in your crazyflies.yaml file as initial guess).
